src/server/rdb_load.cc

```diff
@@ -2049,8 +2051,11 @@ error_code RdbLoaderBase::HandleJournalBlob(Service* service) {
   while (done < num_entries) {
     journal::ParsedEntry entry{};
     SET_OR_RETURN(journal_reader_.ReadEntry(), entry);
+    if (entry.opcode != journal::Op::EXEC)
```

Comment:
redundant line

Reply:
it's not redundant, just messed up formatting :sweat_smile: